portfolio/src/main/webapp/script.js

```diff
@@ -17,4 +17,21 @@ async function showFact() {
   const textFromResponse = await responseFromServer.json();
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = textFromResponse[Math.floor(Math.random() * 4)];
-}
\ No newline at end of file
+}
+
+function loadTasks() {
+  fetch('/list-tasks').then(response => response.json()).then((tasks) => {
+    const taskListElement = document.getElementById('task-list');
```

Comment:
I don't see an element with this id. Is this something you're planning to add later? If so, how did you test that the new code works as intended?

Reply:
I was planning to add, but some parts didn't work so I just deleted